src/tests/ftest/util/general_utils.py

```diff
@@ -608,3 +612,55 @@ def error_count(error, hostlist, log_file):
                 other_error_count += 1
 
     return requested_error_count, other_error_count
+
+
+def get_module_class(name, module):
+    """Get the class object in the specified module by its name.
+
+    Args:
+        name (str): class name to obtain
+        module (str): module name in which to find the class name
+
+    Raises:
+        DaosTestError: if the class name is not found in the module
+
+    Returns:
+        object: class matching the name in the specified module
+
+    """
+    try:
+        name_module = import_module(module)
+        name_class = getattr(name_module, name)
+    except (ImportError, AttributeError) as error:
+        raise DaosTestError(
+            "Invalid '{}' class name for {}: {}".format(name, module, error))
+    return name_class
+
+
+def get_job_manager_class(name, job=None, subprocess=False, mpi="openmpi"):
```

Comment:
Minor: How about staying consistent with the default values for mpi_type? In the apricot/test.py it is defaulted to "mpich" (Lines 373-374)
` manager_mpi_type = self.params.get(
             "job_manager_mpi_type", default="mpich")`
Should help avoid confusion.

Reply:
The "openmpi" default was assigned to align with [this](https://github.com/daos-stack/daos/blob/c4122d1efce4bd4586f135e7fba4a4cefda01c98/src/tests/ftest/util/job_manager_utils.py#L273) default:
```
class Mpirun(JobManager):
    """A class for the mpirun job manager command."""

    def __init__(self, job, subprocess=False, mpitype="openmpi"):
```